zw-knabbel-wp.php

```
@@ -62,6 +62,98 @@ function debug_enabled(): bool {
+
+		$values = array();
+		foreach ( array_slice( $context[ $key ], 0, 20 ) as $value ) {
+			if ( is_scalar( $value ) ) {
+				$values[] = sanitize_key( (string) $value );
+			}
+		}
+		$safe_context[ $key ] = $values;
+	}
+
+	return $safe_context;
+}
```

PHPMD (2.15.0)

[warning] The function prepare_log_context_for_storage() has a Cyclomatic Complexity of 13. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)